pkg/catalogsourceconfig/triggerer.go

```diff
@@ -61,6 +61,20 @@ func (t *triggerer) Trigger(notification datastore.PackageUpdateNotification) er
 
 	allErrors := []error{}
 	for _, instance := range cscs.Items {
+		if instance.Spec.Source != notification.GetOpSrc() {
```

Comment:
Are you introducing this just for debug statements? I don't follow what other purpose it has. If it is just for producing debug logs, please remove it.

Reply:
Then please add a comment that says that. It is not immediately obvious. I am also not a huge fan of littering the code with debug logs that require recompiling to enable. I prefer you carry debug patches locally. So please remove the debug logs.

Reply:
I will remove the debug logs.

Reply:
And add the comment.

Reply:
The debug logs have been removed and a comment has been added.